src/main/java/org/example/controllers/PingController.java

```diff
@@ -0,0 +1,17 @@
+package org.example.controllers;
+
+
+
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@ResponseBody
+public class PingController {
+    @RequestMapping("/ping")

```

Comment:
1. revert to `@GetMapping`
2. remove path "/ping" as you'll move it to the class level (see comment above)